Code/GraphMol/ChemReactions/Wrap/testReactionWrapper.py

```diff
@@ -408,10 +408,10 @@ def test13GetTemplates(self):
     rxn = rdChemReactions.ReactionFromSmarts('[C:1]1[O:2][N:3]1>>[C:1][O:2].[N:3]')
     r1 = rxn.GetReactantTemplate(0)
     sma = Chem.MolToSmarts(r1)
-    self.assertEqual(sma, '[C:1]1-,:[O:2]-,:[N:3]-,:1')
+    self.assertEqual(sma, '[C:1]1[O:2][N:3]1')
     p1 = rxn.GetProductTemplate(0)
     sma = Chem.MolToSmarts(p1)
-    self.assertEqual(sma, '[C:1]-,:[O:2]')
+    self.assertEqual(sma, '[C:1][O:2]')
```

Comment:
This isn't part of the PR, but I can't find in the smarts definition that "[C][C]" == "[C]-,:[C]"

There is a line saying essentially not to specify undefined items (but that's a bracket versus non bracket thing).

Reply:
“A missing bond symbol is interpreted as "single or aromatic".”

Reply:
And, yeah the speed up is because these very common queries are simpler. Way simpler

Reply:
And, yeah the speed up is because these very common queries are simpler. Way simpler